pages/developers/coverage.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: "Code coverage"
+permalink: /developer/coverage
+nav_order: 4
+parent: Developer information
+---
+
+{% include toc.html %}
+
+The "Code coverage" value of a codebase indicates how much of the production/development code is covered by the running unit tests. Maintainers try their best to keep this percentage high, and this process is often automated using tools like `GitHub Actions` and `Codecov`. Hence, code coverage becomes a good metric (not always) to check if a particular codebase is well tested and reliable.
+
+Tools and libraries used to calculate, read, and visualize coverage reports:
+
+- `pytest-cov`: allows developers to calculate and visualize the coverage value
+- `Codecov`: integrates with remote repositories, allowing developers to see and compare coverage value with each CI run
+- `GitHub Actions`: allows users to automatically upload coverage reports to `Codecov`
+
+> ### Are there any alternatives?
```

Comment:
@Saransh-cpp, why is this section indented with ">"? Is this a left-over from a modification or clean-up?

Reply:
The ">" displays them like a suggestion or some additional information -

![image](https://user-images.githubusercontent.com/74055102/184613659-c8179fbf-6deb-48b4-9222-0528461fa012.png)

Reply:
OK, sounds good.